### PR TITLE
Srubenst/synonyms update

### PR DESCRIPTION
**Pull request by blrubenstein**

### CODE REVIEW

File: ui/src/app/views/concept-table/component.tsx [hunk 1]
```diff
@@ -17,10 +19,60 @@ const styles = reactStyles({
...
+    const hasOverflowingChildren = element.offsetHeight < element.scrollHeight ||
+      element.offsetWidth < element.scrollWidth;
+    this.setState({willOverflow: hasOverflowingChildren});
+  }
 
-export class ConceptTable extends React.Component<{concepts: Concept[];
-  loading: boolean; placeholderValue: string, getSelectedConcepts: Function,
-  setSelectedConcepts: Function; },
+  render() {
+    return <div style={{display: 'flex'}}>
```
Comment (NehaBroad) [inline ui/src/app/views/concept-table/component.tsx]: not: we can extrapolate props and state here.

File: ui/src/app/views/concept-table/component.tsx [hunk 2]
```diff
@@ -17,10 +19,60 @@ const styles = reactStyles({
...
+      <div style={{minWidth: '170px', maxWidth: '170px',
+        fontStyle: 'italic',
```
Comment (NehaBroad) [inline ui/src/app/views/concept-table/component.tsx]: Styles with more than 2 attributes can be moved under styles const

File: ui/src/app/views/concept-table/component.tsx [hunk 3]
```diff
@@ -17,10 +19,60 @@ const styles = reactStyles({
   }
 });
 
+export class SynonymsObject extends React.Component<{},
```
Comment (NehaBroad) [inline ui/src/app/views/concept-table/component.tsx]: since there is no property we can have const SynonymsObject instead of class

Comment (blrubenstein) [reply ui/src/app/views/concept-table/component.tsx]: Resolved offline.

**Pull request merged**